src/js/theme.js

```
@@ -0,0 +1,138 @@
+      let dateStr;
+      try {
+        dateStr = m.date.toLocaleDateString('en-US', { month: 'short', day: 'numeric', year: 'numeric' }).toUpperCase();
+      } catch (err) {
+        console.warn('[Timeline] Date formatting failed for:', m.date, err);
+        dateStr = m.date.toISOString().slice(0, 10);
+      }
+
+      if (isActive && !allPast) {
+        return `<div class="flex gap-3 sm:gap-4">
+          <div class="flex flex-col items-center"><div class="w-3 h-3 rounded-full bg-primary ring-4 ring-orange-100 dark:ring-orange-950/40 pulse-dot"></div>${connector}</div>
+          <div><p class="text-[10px] font-bold text-primary">${escapeHtml(dateStr)}</p>
```

ESLint

[error] 'escapeHtml' is not defined.

(no-undef)